managing_providers/_topics/embedded_workflows.md

```diff
@@ -0,0 +1,433 @@
+## Embedded Workflows
+{: #embedded_workflows}
```

Comment:
@rwellon What is this symbol?  When I view it rendered, it shows up.

![manageiq-documentation_managing_providers__topics_embedded_workflows_md_at_10f4ce35938dffb91aef18139c2dcca861de5869_·_ManageIQ_manageiq-documentation](https://github.com/ManageIQ/manageiq-documentation/assets/52120/d6bf57cc-507a-4df4-94e7-4341cba068fd)

Reply:
In the downstream IA docs we use the anchors for adding links to the headings within topics. I notice in ManageIQ docs this is not used. I removed this anchor.